src/renderer/renderScene.ts

```diff
@@ -113,11 +120,9 @@ export function renderScene(
   context.save();
   scaleContextToZoom();
   translateContextToZoom();
-  context.translate(sceneState.scrollX, sceneState.scrollY);
   visibleElements.forEach(element => {
     context.save();
-    context.translate(element.x, element.y);
```

Comment:
Here you are removing scroll and element position based translations.

Reply:
That's correct, I' doing it inside the `renderElement()`